src/components/SemanticViewer.jsx

```diff
@@ -256,31 +257,42 @@ class SemanticViewer extends Component {
 
 	onClickDataset = async (semantic) => {
```

Comment:
It might a typo when copied from the dataviewer logic.  `onClickDataset` should be `onClickSemanticsEntry` or something to avoid confusion...
you might want to do a search overall in this file to see if there is anything still named `dataset`